docs/architecture/assessment-runtime/plugin-registry.md

```diff
@@ -0,0 +1,51 @@
+# Plugin Registry
+
+:::caution
+This initial design assumes that the plugin architecture will be built on WASM technology. Things might change after the initial POC.
+:::
+
+### 1. **Plugin Repository on GitHub**:
+The designated GitHub repository stores all plugins as WebAssembly (WASM) modules. Each plugin is versioned and audited for security.
```

Comment:
nit: I think having plugins in a registry would be a more standard approach, and allow for better security interfaces (people could mirror the registry, run whatever the tools, we could sign the attestations for Supply Chain, provide SBOMs, etc)

Reply:
I have another design that showcases a full-blown registry, but you know, for the MVP, I tried to keep it simple. :) What do you think?

Reply:
LGTM :D